dropdown.vue

```diff
@@ -1,8 +1,8 @@
 <template>
 	<div
 		class="dropdown"
-		@click.stop="toggleDropdownOptions"
-		:class="{ expanded: isExpanded }"* 
+		@click.stop="checkDropdownHandler"
+		:class="{ expanded: isExpanded, disabled: config.disabled }"

```

Comment:
can you decouple this class object to a computed `dropdownClasses` property that returns that object for more clarity and separation of concerns?